ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -264,6 +306,19 @@ open class CommandBar: UIView {
         containerMaskLayer.locations = locations.map { NSNumber(value: Float($0)) }
     }
 
+    private func updateButtonTokens() {
+        leadingCommandGroupsView.updateButtonGroupViews()
```

Comment:
I am not sure that exposing `updateButtonGroupVIews` as public and calling it here is safe. The implementation of `updateButtonGroupVIews` calls `updateItemsToButtonsMap` to refresh the items-to-buttons map and then creates new buttons for each item, but the newly created buttons are never added to the stack view. This could cause an issue where the state of the `CommandBarItem` is not consistent the corresponding `CommandBarButton`. If this sort of update behavior is needed, I would suggest trying to expose `updateButtonsShown ` instead as it will update the stack view after calling `updateButtonGroupVIews`.